tests/vmi_networking_test.go

```diff
@@ -756,11 +756,11 @@ var _ = Describe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 					ipv4NetworkCIDR = api.DefaultVMCIDR
 				}
 				By("Checking ping to gateway")
-				pingVirtualMachine(serverVMI, gatewayIPFromCIDR(ipv4NetworkCIDR), "\\$ ")
+				testPingFromVMI(serverVMI, gatewayIPFromCIDR(ipv4NetworkCIDR), "\\$ ")
```

Comment:
Since the rror was remove from `pingVirtualMachine` please add here the error handling.